checker-qual/src/main/java/org/checkerframework/checker/formatter/qual/UnknownFormat.java

```diff
@@ -18,8 +18,8 @@
  * <p>A type annotation indicating that the run-time value might or might not be a valid format
  * string.
  *
- * <p>This annotation may not be written in source code; it is an implementation detail of the
- * checker.
+ * <p>This annotation may not be explicitly written in source code; it is an implementation detail
+ * of the checker.
```

Comment:
may not == "it has the ability but should not"

it's usually not necessary writing this, it's implementation details. but you can.